app/views/podcast_engagement/show.html.erb

```diff
@@ -4,47 +4,38 @@
 
 <%= form_with(model: @podcast, url: podcast_engagement_path, html: {autocomplete: "off"}, data: {controller: "unsaved", unsaved_changed_value: @podcast.changed?, unsaved_confirm_value: t(".confirm")}) do |form| %>
 <div class="container my-4">
-    <div class="d-grid align-items-center">
-        <div class="row my-4">
-            <div class="col-6">
-                <div class="mb-4 prx-field-group">
-                    <div class="input-group">
-                        <div class="form-floating">
-                            <%= form.text_field :donation_url %>
-                            <%= form.label :donation_url %>
-                        </div>
-
-                        <div class="input-group-text"> <span class="material-icons">link</span> </div>
+    <div class="row my-4">
+        <div class="col-6">
+            <div class="col-12 mb-4">
```

Comment:
May be a good @brandonhundt question; tried to follow the pattern of other forms, but the `col-6` kept the form/input fields sized correctly, trying to wrap it in the `col-12` instead doesn't.
Wouldn't the `col-12` denote the entire width of the wrapping container, and `col-6` use half of that?

Reply:
Try this starting at line 7...

```
<div class="col-lg-8">
  <div class="row">
    <div class="col-12 mb-4">
    
```

then the side bar would be:

```
<div class="col-lg-4">
  <div class="row">
```